rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/interactor/CovidPatientHistoryActivityInteractorTest.java

```diff
@@ -0,0 +1,140 @@
+package io.ona.rdt.interactor;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.powermock.reflect.Whitebox;
+import org.robolectric.util.ReflectionHelpers;
+import org.smartregister.domain.Event;
+import org.smartregister.domain.Obs;
+import org.smartregister.domain.db.EventClient;
+import org.smartregister.repository.EventClientRepository;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import edu.emory.mathcs.backport.java.util.Collections;
+import io.ona.rdt.domain.PatientHistoryEntry;
+import io.ona.rdt.repository.PatientHistoryRepository;
+import io.ona.rdt.robolectric.RobolectricTest;
+import io.ona.rdt.util.FormKeyTextExtractionUtil;
+
+public class CovidPatientHistoryActivityInteractorTest extends RobolectricTest {
```

Comment:
Please rename the methods here to follow the convention

Reply:
test methods renamed.